features/personne/filiere_crud.feature

```diff
@@ -32,13 +33,21 @@ Feature: Filiere
     And I should see "Filière modifiée"
     And I should see "Testing science"
 
+  Scenario: I can not delete a Filiere with members
+    Given I am logged in as "admin"
+    Given I am on "/personne/filiere/modifier/1"
+    Then the response status code should be 200
+    And I press "Supprimer la filière"
+    Then the url should match "/personne/poste"
+    And I should see "Impossible de supprimer une filiere ayant des membres."
+
   # The "@dropSchema" annotation must be added on the last scenario of the feature file to drop the temporary SQLite database
   @dropSchema
   Scenario: I can delete a Filiere
     Given I am logged in as "admin"
-    Given I am on "/personne/filiere/modifier/1"
+    Given I am on "/personne/filiere/modifier/6"
```

Comment:
You're using MySQL which is a bit of a shame, but if you ever switch to PostgreSQL (which is better in every imaginable ways), the auto-generation of ids is made by sequences. For example for `user`, if you have auto-generated int IDs for it, it would be provided by a sequence `user_id`. All the sequence does is creating a new value and incrementing itself. While this seem a bit more complicated, it allows you to easily reset it between each scenarios, which means you have a better control on the IDs used

Reply:
MySQL is  suited for the Jeyser CRM project. 

Moreover we will not switch because:

 * It would break [every already running installation](https://github.com/n7consulting/Incipio/issues/76) (or create a cumbersome migration for little to no additionnal business value)
 * It makes imports easier because Siaje also uses MySQL.
 * Tooling is better (pgadminTools << PhpMyAdmin)
 * It would break all the previous migrations

Reply:
If you're using InnoDB it should be possible to reset the auto-increment with:

```SQL
ALTER TABLE tablename AUTO_INCREMENT = 1
```

which would be an alternative solution. Switching of DB is never easy so I don't recommend to invest too much time on it. It might however be worth checking if the migrations have any MySQL specific features or not. It's far easier to switch from MySQL to PostgreSQL than vice-versa as MySQL is _much_ more limited than Postgres.

Regarding the tooling I don't have much of an opinion as I'm really fine using the CLI client, it's something you need do at some point anyway but I understand phpmyadmin can be enough for limited usage.